docs/commands/mftf.md

```diff
@@ -173,7 +173,7 @@ Complex configuration to generate a few non-suite tests, a single test in a suit
 The command that encodes this complex configuration:
 
 ```bash
-vendor/bin/mftf generate:tests --tests "{\r\n\"tests\":[\r\n\"general_test1\",\r\n\"general_test2\",\r\n\"general_test3\"\r\n],\r\n\"suites\":{\r\n\"sample\":[\r\n\"suite_test1\"\r\n],\r\n\"sample2\":null\r\n}\r\n}"
+vendor/bin/mftf generate:tests --tests "{"tests\":["general_test1\","general_test2\","general_test3\"],"suites\":{"sample\":["suite_test1\"],"sample2\":null}}"
```

Comment:
I think you accidentally removed some backslashes that escape the inner double quotes.

It should be:

```
"{\"tests\":[\"general_test1\",\"general_test2\",\"general_test3\"],\"suites\":{\"sample\":[\"suite_test1\"],\"sample2\":null}}"
```

Reply:
Does this look correct now?

Reply:
It is correct now.

But it just occurred to me we could also simplify it further by using single quotes outside and double quotes inside:

```
'{"tests":["general_test1","general_test2","general_test3"],"suites":{"sample":["suite_test1"],"sample2":null}}'
```

It's up to you. Whichever you prefer.

Reply:
Even better. I am happy to be rid of them.